meteor/client/lib/__tests__/rundown.test.ts

```diff
@@ -245,7 +245,7 @@ describe('client/lib/rundown', () => {
 				const outputLayerIds = Object.keys(showStyleBase.outputLayers)
 
 				const playlistActivationId = protectString('mock_activation_0')
-				mockRundownPlaylistsCollection.update(playlistId, {
+				mockRundownPlaylistsCollection.update(unprotectString(playlistId), {
```

Comment:
This seems like a step backwards in type safety, is it possible to keep the protectedString here?

Reply:
Not easily. this `mockRundownPlaylistsCollection` is of type `Meteor.Collection`, so we would either need to return a wrapped type for these mock collections to be used here, or override the typings of the collection. As this is test code, and only a couple of places, I am not sure it is worth the cost to attempt either route.

Fun fact, the old query is not at all type safe. I can change it to `mockRundownPlaylistsCollection.update(showStyleBase._id, {` and typescript does not care. I think because its using the loose mongo queries. Only now it is being strict and rejecting the `ProtectedString` as something that isnt compatible with a `MongoQuery<T>`